bundles/org.openhab.binding.magentatv/src/main/java/org/openhab/binding/magentatv/internal/.gitignore

```diff
@@ -0,0 +1 @@
+/.DS_Store
```

Comment:
remove this file, the DS_Store is already in the gitignore of the whole repo

Reply:
done